app/controllers/administrate/application_controller.rb

```diff
@@ -3,53 +3,42 @@ class ApplicationController < ActionController::Base
     protect_from_forgery with: :exception
 
     def index
-      search_term = params[:search].to_s.strip
-      resources = Administrate::Search.new(scoped_resource,
+      @search_term = params[:search].to_s.strip
```

Comment:
I'm concerned here that we're making a lot of variables available with these changes (and there's quite a few of them). Are we able to pull this off without making these changes?